style.css

```diff
@@ -1,3 +1,429 @@
-h1 {
-  color: green;
+body {
+  background: #e5e5e5;
+  background-image: ./hello/images/header-shapes.png;
```

Comment:
- [ ] This line of code seems not working as it is in the wrong format. Kindly remove this or fix this issue.